lbry/wallet/orchstr8/node.py

```diff
@@ -440,3 +458,139 @@ def decode_raw_transaction(self, tx):
 
     def get_raw_transaction(self, txid):
         return self._cli_cmnd('getrawtransaction', txid, '1')
+
+
+class HubProcess(asyncio.SubprocessProtocol):
+
+    IGNORE_OUTPUT = [
+        b'keypool keep',
+        b'keypool reserve',
+        b'keypool return',
+    ]
```

Comment:
this list probably needs to be updated for hub output to ignore

Reply:
removed